samples/prerequisites.md

```diff
@@ -1,23 +1,51 @@
-If you're new to .NET make sure to visit the [official starting page](http://dotnet.github.io). It will guide you through installing pre-requisites and building your first app.
+If you're new to .NET, make sure to visit the [official starting page](http://dotnet.github.io). It will guide you through installing pre-requisites and building your first app.
 If you're already familiar with .NET make sure you've [downloaded and installed the .NET 5 SDK](https://www.microsoft.com/net/download/core).
 
-The following pre-requisites need to be installed for building .NET 5 projects with CoreRT:
+The following pre-requisites need to be installed for building .NET 5 projects with CoreRT.
 
 # Windows
 
-* Install [Visual Studio 2019](https://visualstudio.microsoft.com/vs/community/), including Visual C++ support.
+## Install [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/) with Visual C++ Build Tools
+
+The free [Community](https://visualstudio.microsoft.com/vs/community/) edition or higher is recommended.
+If you wish to save disk space and do not need Visual Studio IDE, here is the minimal [Build Tools](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019)
+edition installation required. First download the [bootstrapper](https://aka.ms/vs/16/release/vs_buildtools.exe) executable.
+On Windows 10 version 1803 or later you may use the `curl` tool:
+
+```cmd
+curl -L https://aka.ms/vs/16/release/vs_buildtools.exe -o vs_buildtools.exe
+```
+
+Then launch the bootstrapper passing the installation path and the two required components (requires elevation):
+```cmd
+vs_buildtools.exe --installPath C:\VS2019 --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64 Microsoft.VisualStudio.Component.Windows10SDK.19041 --passive --norestart --nocache --lang en-US
```

Comment:
I have just run it and it did not install anything into `c:VS2019`. Does this modify your existing VS installation if there is one already?

Reply:
Need to remove `--lang en-US` (it is used for creating a layout only). Thanks for catching this.